project-base/src/Controller/Front/CartController.php

```diff
@@ -332,4 +339,34 @@ public function deleteAction(Request $request, $cartItemId)
 
         return $this->redirectToRoute('front_cart');
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @param int $cartItemId
+     */
+    public function deleteAjaxAction(Request $request, $cartItemId)
+    {
+        $cartItemId = (int)$cartItemId;
+        $token = new CsrfToken('front_cart_delete_' . $cartItemId, $request->query->get('_token'));
+
+        if ($this->get('security.csrf.token_manager')->isTokenValid($token)) {
+            try {
+                $this->cartFacade->deleteCartItem($cartItemId);
+            } catch (\Shopsys\FrameworkBundle\Model\Cart\Exception\InvalidCartItemException $ex) {
+                return $this->json([
+                    'success' => false,
+                    'errorMessage' => $ex->getMessage(),
```

Comment:
Error message for the customer will be always in English here (no translation is used).
And it's not possible to just `t(` the `getMessage` method as a variable is part of a message. 

I suggest to replace it with some generic translated sentence. 
(Don't forget to dump translations afterwards)

Reply:
Ok, I replaced it as string who is used some lines upper, so I dont have to dump translations, right?